src/main/java/com/example/AkkaMainSystem.java

```diff
@@ -24,11 +24,18 @@ public Receive<Create> createReceive() {
 
     private Behavior<Create> onCreate(Create command) {
         //#create-actors
-        ActorRef<ExampleActor.Message> a = this.getContext().spawn(ExampleActor.create("Alice"), "alice");
-        ActorRef<ExampleTimerActor.Message> b = this.getContext().spawn(ExampleTimerActor.create(), "timeractor");
-        //#create-actors
+        ActorRef<Queue.Message> queue = this.getContext().spawn(Queue.create(), "queue");
+        ActorRef<Scheduler.Message> scheduler = this.getContext().spawn(Scheduler.create(queue ), "scheduler");
+
+        for (int i = 0; i < 5; i++) {
+            ActorRef<Task.Message> task = this.getContext().spawn(Task.create(scheduler, queue, i+1), "task_" + (i+1));
+            queue.tell(new Queue.AddTask(task));
+        }
+
```

Comment:
Hier rufen wir 5 mal den Task auf, sollen wir das noch irgendwie dynamisch machen (anzahl der Zahlen in der Liste)?

Reply:
ich habe das auch erst nach dem Commit gemerkt. Ich habe bei mir auf 20 gestellt und es scheint zu funktionieren